### DISCUSSION TITLE
DatePicker选择周展示错误

### DISCUSSION THREAD

**Discussion by yuruwang** [ant-design/ant-design#28526] [Q&A] [answered]
我在用DatePicker选择周时，发现当鼠标hover到2019.30和2019.31号时，会显示成2019年第一周，如下图：
![image](https://user-images.githubusercontent.com/31492974/103073849-4fa52000-4603-11eb-97c0-f3d6f0a7e9e5.png)

但是在antd官网上并没有复现出这个错误。我的配置只有picker: 'week'，format: 'YYYY-wo'，以及根据官网文档设置了locale。不知道有没有人也遇到过这种类型的问题...

使用的版本：4.9.4

**Comment by afc163** [comment, marked as answer]
`gggg-Wo`

**Reply by yuruwang** [reply]
解决了，感谢～
